### PR TITLE
ADVI stepsize sequence parameter (eta) adaptation

### PR DESCRIPTION
**Pull request by akucukelbir**
See https://github.com/stan-dev/stan/pull/1699

### CODE REVIEW

File: src/cmdstan/command.hpp [hunk 1]
```diff
@@ -872,16 +880,16 @@ namespace stan {
...
-                     eta,
-                     base_rng,
                      eval_elbo,
                      output_samples,
                      &std::cout,
                      output_stream,
                      diagnostic_stream);
-          cmd_advi.run(tol_rel_obj, max_iterations);
+          cmd_advi.run(eta, adapt_engaged, adapt_iterations,
+            tol_rel_obj, max_iterations);
```
Comment (bob-carpenter) [inline src/cmdstan/command.hpp]: This isn't critical, but here and on line 861, it's conventional to start the next line with the arguments under the last line.  cpplint's OK with your way, but it's easier to see the function if you have the room to indent.

**Comment by bob-carpenter** [comment]
This looks OK --- feel free to clean up formatting or not.  I'm going to take a pass over all of this after the dust settles and try to clean up style and add full comments.

**Comment by syclik** [comment]
Jenkins, retest this please.

**Comment by syclik** [comment]
@akucukelbir, if there's an open issue, can you close it for me? I don't know what it would be called.

**Comment by akucukelbir** [comment]
i'll take a look. i don't think there is. we're using https://github.com/stan-dev/stan/pull/1590 as a blueprint. (really suboptimal, i know.) i'll try to split the remaining things into issues and follow the naming convention for the upcoming pull requests.

**Pull request merged**